cfme/tests/control/test_actions.py

```diff
@@ -188,20 +121,27 @@ def compliance_condition(appliance):
         expression="fill_tag(VM and Instance.My Company Tags : Service Level, Gold)"
     )
     yield _compliance_condition
-    _compliance_condition.delete()
+    _compliance_condition.delete_if_exists()
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture
 def compliance_policy(vm_name, policy_name, appliance):
```

Comment:
Question: 
- Why do we need to change the fixture scope of these fixtures?

Reply:
This test module was originally designed to use a single vm for all of the action tests, while this is optimal in terms of speed, it makes these tests extremely flakey. I.e. if a single test fails then I was seeing all the tests fail. It is clearer to use a different VM for each test, and much easier to address test failures.